OCP-Infrastructure/helm/hogarama/charts/mongodb/values.yaml

```diff
@@ -0,0 +1,11 @@
+---
+credentials:
```

Comment:
are those dummy vaules?

Reply:
yes, those are your dummy values you crated for the templating test on travis